README.md

```diff
@@ -52,6 +52,23 @@ These examples provide quick walkthroughs to get you up and running with Amazon
 - [IP Insights for suspicious logins](introduction_to_amazon_algorithms/ipinsights_login) shows how to train IP Insights on a login events for a web server to identify suspicious login attempts.
 - [Semantic Segmentation](introduction_to_amazon_algorithms/semantic_segmentation_pascalvoc) shows how to train a semantic segmentation algorithm using the Amazon SageMaker Semantic Segmentation algorithm. It also demonstrates how to host the model and produce segmentaion masks and probability of segmentation.
 
+### Amazon SageMaker RL 
+
+The following examples provide quick examples each demonstating different capabilities of Amazon SageMaker RL. 
```

Comment:
"examples" repeated. Perhaps, change with:

The following provide examples demonstating different capabilities of Amazon SageMaker RL.